.travis.yml

```diff
@@ -23,7 +24,7 @@ before_install:
   - cd WcaOnRails/
 before_script:
   - bundle exec rake db:reset RAILS_ENV=test
-sudo: false
+sudo: required
```

Comment:
Damn! Can't win =( Sounds like we'll have to go containerless, and hope that Travis gets their act together.